yieldies/fees.md

```diff
@@ -1,10 +1,10 @@
 # Fees
 
-There are a few places that fees are involved in Yieldies that user's should be aware of. 
+There are a few places that fees are involved in Yieldies that users should be aware of. 
```

Comment:
Syntactic nitpic spotted on the original PR which got merged too fast, IMO reads better than the double "that"
```suggestion
There are a few places where fees are involved in Yieldies that users should be aware of. 
```